docs/business-apps/introduction-to-sharepoint-business-process-integration.md

```diff
@@ -8,39 +8,39 @@ localization_priority: Priority
 
 # SharePoint custom forms and automated business workflow
```

Comment:
@jackwi-alt thinking of it now, how about the title: "Business Apps and Business Process Automation" ? That keeps it generic to the feature.

Reply:
Change made. Please review and merge my PR into master, if approved.